Script/AtomicNET/AtomicNET/Atomic2D/PhysicsWorld2D.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace AtomicEngine
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct PhysicsRaycastResult2D
+    {
```

Comment:
Yes, I agree, but as I said in the issue, I really think we should have consistency between the 2D and the 3D physics API, so it's easy for people to go back and forth. I really like how in Unity for example they are consistent, except for some peculiarities and of course one axis :P. That requires some bigger changes though and for the time being I guess this is enough for some use cases.